.buildkite/pipeline.yml

```diff
@@ -30,10 +30,18 @@ steps:
       cp gradle.properties-example gradle.properties
       ./gradlew test
 
+  - label: "API verification"
+    key: "api"
+    plugins: [$CI_TOOLKIT]
+    command: |
+      cp gradle.properties-example gradle.properties
+      ./gradlew :experimentation:apiCheck
```

Comment:
Praise (❤️): Awesome, love it, thanks for adding the `binary-compatibility-validator` plugin into this repo, let's start using this more and more within this project and other repos as well! 🥇